### PR TITLE
W3C Calendar with fullcalendar library

### PR DESCRIPTION
**Pull request by danielpeintner**
TODOs:
* [ ] use online ICS file instead of local one (easier for now w.r.t. testing)
* [x] add complete list of timezones in TZ selector
* [x] listen onload events to collect W3C events
* [ ] start website with local timezone as we used to have?
* [ ] rephrase intro text "We have grouped our weekly meetings in the table below...."
* [x] limit view to weekdays only

ISSUES:
* [ ] changing week to month view does not reload timezone correctly
* [ ] moment(..) always reports local time
* [ ] repeat events not shown after TZ changer
* [x] avoid calendar entry being editable

fixes https://github.com/w3c/wot-marketing/issues/268

### CODE REVIEW

**Comment by egekorkan** [comment]
This is actually pretty good for a first try. I agree with the improvements above. Also not showing the useless days (Friday, Saturday, Sunday) and useless hours (where there are no meetings) would be nice improvements that can be done in another PR

**Comment by danielpeintner** [comment]
> Also not showing the useless days (Friday, Saturday, Sunday)

Partially done. It does no longer show weekends

File: docs/activities/meetings/index.html [hunk 1]
```diff
@@ -5,281 +5,187 @@
...
-            <td style="background-color: #EEEEEE; text-align:center;">13:00 - 14:00</td>
-            <td style="background-color: #EEEEEE; text-align:center;">14:00 - 15:00</td>
-            <td style="background-color: #EEEEEE; text-align:center;">15:00 - 16:00</td>
-            <td style="background-color: #EEEEEE; text-align:center;">16:00 - 17:00</td>
-            <td style="background-color: #EEEEEE; text-align:center;">17:00 - 18:00</td>
-            <td style="background-color: #EEEEEE; text-align:center;">18:00 - 19:00</td>
-        </tr> -->
+<script>
+	var icsEvents = [];
+	var initialTimeZone = "America/New_York";
```
Comment (egekorkan) [inline docs/activities/meetings/index.html:25]: I think that this can use the initTimezone function that is deleted in this PR

**Comment by egekorkan** [comment]
I think that the library you have used before (https://gitlab.nomagic.uk/popi/js_calendar_from_ics/-/tree/master/) is doing exactly that no?

**Comment by danielpeintner** [comment]
> I think that the library you have used before (https://gitlab.nomagic.uk/popi/js_calendar_from_ics/-/tree/master/) is doing exactly that no?

Maybe, did not look into it more closely... the library we are using (https://cdnjs.cloudflare.com/ajax/libs/ical.js/1.5.0/ical.js) does also something along these lines but I guess it is better if we can rely on the library. Hence I opened https://github.com/fullcalendar/fullcalendar/issues/7364

For the time being I think we can also have a simple solution for now.

**Comment by egekorkan** [comment]
Call of 06.06:
- This is a simple approach that uses ics and not the W3C API. So we do not have a strong dependency on W3C systems.
- We should not redo the work of W3C Systems team

**Comment by egekorkan** [comment]
Call of 13.06: Let's wait 2 weeks to see any progress on the abovementioned issue. Otherwise, we can switch to https://gitlab.nomagic.uk/popi/js_calendar_from_ics/-/tree/master/ and do the timezone changes before passing it to the fullcalendar.

**Comment by egekorkan** [comment]
Call of 11.07: @danielpeintner will try with the [js_calendar_from_ics](https://gitlab.nomagic.uk/popi/js_calendar_from_ics/-/tree/master/) from above. If that works, no manual timezone conversion is needed. https://gist.github.com/danielpeintner/c2a744aa2c37574d7e2e16686e64f84d can be used as the starting point.

**Comment by danielpeintner** [comment]
I quickly looked into the library and I think it works somehow.. anyhow.. the library seems to have issues with timezone again 🤷‍♂️

https://gitlab.nomagic.uk/popi/js_calendar_from_ics/-/blob/master/js/ical_fullcalendar.js#L10-13

**Comment by egekorkan** [comment]
Call of 18.07:
- We need to transform strings like `DTSTART;TZID=America/New_York:20230620T080000` to full calendar json using the script above. Full Calendar requires `start: '2018-09-01T12:30:00+XX:XX'`. Since we know that all W3C events are in East Coast tz, we can hardcode the `+XX:XX` to UTC -5. Then, as shown in the demo (https://fullcalendar.io/docs/timeZone-demo), we can transform that to the local timezone of the browser client.

**Comment by egekorkan** [comment]
Now fixed via #495

**Pull request closed without merging**